Kyle

kldaly5430 opened the pull request:
I believe my page is done and ready

markzz commented:
Produce new commits that do the following...

- Remove html files (they are extraneous and unnecessary)
- Remove extraneous HTML in the php file that is for the header and footer.
- Move images to web/html/images and make sure you change the references to them as well

If all of those are done, then it will be good to go!

kldaly5430 commented:
I remove the html file and moved the image. I think I removed all the extraneous html from the header. The footer I didn't see any extraneous html, but I could be over looking it. So just tell me if there is still some.

markzz commented:
Since some incompatibilities with master, I will merge this tomorrow by hand.
